constants/constants.go

```diff
@@ -2,3 +2,77 @@ package constants
 
 // CustomDateFormat is a date format used in the application to parse date string/object into a usable format
 const CustomDateFormat = "2006-01-02"
+
+// constants defined to use in ActionItemTypeMap
+const (
```

Comment:
Let's create a custom type for these constants and use them everywhere. See how we have done it in the application.